analyzere/__init__.py

```diff
@@ -2,6 +2,7 @@
 password = ''
 base_url = 'http://localhost:8000/'
 upload_poll_interval = 0.1
+upload_chunk_size = 16 * (1 << 20)
```

Comment:
`one_megabyte = 2**20`
`upload_chunk_size = 16 * one_megabyte`

if you want to be very clear